Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp

```diff
@@ -1155,10 +1164,17 @@ void CoordinatedGraphicsLayer::updateContentBuffers()
     }
 
     if (!m_needsDisplay.completeLayer) {
-        for (auto& rect : m_needsDisplay.rects)
-            layerState.mainBackingStore->invalidate(enclosingIntRect(rect));
-    } else
+        for (auto& rect : m_needsDisplay.rects) {
+            auto rr = enclosingIntRect(rect);
```

Comment:
Use better name for the variable than `rr`

Reply:
done